pom.xml

```diff
@@ -4,7 +4,7 @@
   <groupId>com.findologic</groupId>
   <artifactId>xml-export</artifactId>
   <packaging>jar</packaging>
-  <version>1.5.0-SNAPSHOT</version>
+  <version>1.6.0-SNAPSHOT</version>
```

Comment:
@howard What is the current way to get this version to nexus, so that this build http://10.8.0.1:8081/job/xml-validator_Merge-Request/42/ can pass. Looks like there is no build job for this, that will do this. Do you always manually push this? I think we already had issues with that in the past, but i can't find any notes.